packages/aws-rfdk/docs/upgrade/upgrading-1.5.md.yaml

```diff
@@ -0,0 +1,13 @@
+# Upgrading to RFDK v1.5.x or Newer
+
+## Updating DocumentDB
+
+Please backup your existing database before upgrading to RFDK 1.5.x.
```

Comment:
Maybe we should also say explicitly that deployment will fail unless you manually upgrade your database?